### PR TITLE
feat(autorefresh): Add setting to always autorefresh the monitoring a…

### PR DESCRIPTION
**Pull request by ioanpocol**
…nd search lists (SDNTB-520)

### CODE REVIEW

File: scripts/apps/monitoring/directives/MonitoringGroup.js [hunk 1]
```diff
@@ -412,6 +412,10 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
                 }
 
                 return apiquery(criteria, true).then((items) => {
+                    if (config.features.autorefresh) {
```
Comment (petrjasek) [inline scripts/apps/monitoring/directives/MonitoringGroup.js]: I think we should give it some more specific name, in settings it's hard to guess what is autorefreshed

Comment (ioanpocol) [reply scripts/apps/monitoring/directives/MonitoringGroup.js]: autorefreshLists?

Comment (petrjasek) [reply scripts/apps/monitoring/directives/MonitoringGroup.js]: yep, or monitoringAutorefresh, but it's not only in monitoring right?

Comment (ioanpocol) [reply scripts/apps/monitoring/directives/MonitoringGroup.js]: is it in search too

Comment (petrjasek) [reply scripts/apps/monitoring/directives/MonitoringGroup.js]: maybe autorefreshContent then?

**Pull request merged**